content/02.body.md

```diff
@@ -89,17 +88,22 @@ We provide documentation for parameter choices depending on the type of network
 The original algorithm and our proposed modification are given in Figure {@fig:algo}.
 
 ![
```

Comment:
I am undecided about whether we want a newline here. I think it's cleaner but should check that leading whitespace is stripped.